### PR TITLE
add a storage tank furniture and the associated recipe.

### PR DESCRIPTION
**Pull request by waxcatape**
#### Summary
Content "add a new large opaque 1500L/400 gal standing tank."

#### Purpose of change

The existing standing tank is described as huge despite the fact that it's only 300L in volume. That corresponds to a cylinder about 60 cm in diameter and 1 m in length/height. Each tile is 1 m^2 in area, and characters of any height, which can potentially reach 3 m, can move around indoors.
This implies the lower bound on the volume of each tile at 3 m^3. Then the standing tank only uses 1/10th of the volume. It is more space efficient to just pile up smaller tanks or drums than build or use a standing tank.

The recipe for the standing tank doesn't make too much sense either. We throw together 4 60L tanks to somehow get 5 60L tanks worth of volume in the final product.

#### Describe the solution

Added a new larger tank furniture. It has the volume of 1500L, which represents a cylinder 1 m in diameter and 2 m in height (1570L to be precise, but the tank should probably have some bulk to it). As the new tank is fairly big, it obscures vision. Added a construction recipe for the new tank, based on the recipe for a 200L steel drum, but with requirements scaled with the surface area of the new tank. The time increase in the construction recipe is arbitrary, just felt about right.

#### Describe alternatives you've considered

1) Leave everything as is.

2) Substitute the existing standing tank with the new tank. Decided against it because it would probably break mapgen.

3) Change the standing tank volume and recipe. I'm not sure how that would play with fully or partially filled tanks. 

4) Add a smaller tank instead, maybe in the 600 - 1200L range.

5) Add some other type of furniture liquid storage, like 4, 8, or 12 200L steel drums banded together with wire and stacked on top of one another.

#### Testing

1) Created a new world.

2) Debug spawned in the new tank furniture.

3) Filled it with fuel.

4) Used the new construction recipe to build another tank.

### CODE REVIEW

File: data/json/construction.json [hunk 1]
```diff
@@ -2508,6 +2508,20 @@
     "pre_special": "check_empty",
     "post_terrain": "f_standing_tank"
   },
+  {
+    "type": "construction",
+    "id": "constr_storage_tank",
+    "group": "build_storage_tank",
+    "//": "a huge metal tank, useful for holding very large volumes of liquids",
```
Comment (Paprika-H) [inline data/json/construction.json:2515]:
```suggestion
    "//": "a huge metal tank, useful for holding very large volumes of liquids.",
```

File: data/json/furniture_and_terrain/furniture-storage.json [hunk 2]
```diff
@@ -1137,6 +1137,29 @@
       "items": [ { "item": "scrap", "count": [ 8, 32 ] }, { "item": "water_faucet", "prob": 50 } ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_storage_tank",
+    "name": "bulk storage tank",
+    "description": "A gigantic tank that can store all of the liquid. All of it.",
```
[the remark this answers is not in the document]
Comment (waxcatape) [reply data/json/furniture_and_terrain/furniture-storage.json]: Thank you!

File: data/json/furniture_and_terrain/furniture-storage.json [hunk 3]
```diff
@@ -1137,6 +1137,29 @@
       "items": [ { "item": "scrap", "count": [ 8, 32 ] }, { "item": "water_faucet", "prob": 50 } ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_storage_tank",
+    "name": "bulk storage tank",
+    "description": "A gigantic tank that can store all of the liquid.  All of it.",
```
[the remark this answers is not in the document]
Comment (irwiss) [reply data/json/furniture_and_terrain/furniture-storage.json:1144]: That's hardcoding units though, so people who chose to see distances measured in some dude's feet will have to see liters/meters.

May be still keep a generic description, like the one in PR, or even more generic - "A watertight storage tank" but display capacity in user's units with code in the examine menu?

That said may be displaying capacity would be a good idea regardless of this PR 🤔

[the remark this answers is not in the document]
Comment (Fris0uman) [reply data/json/furniture_and_terrain/furniture-storage.json:1144]: Ah yes I didn't think of the unit issue, my main point is that "gigantic" might a bit too much to dscribe a 2 x 1 m cylinder.

**Comment by mqrause** [comment]
I'm pretty sure a tank furniture with over 1000L capacity isn't really possible right now, since 1000L is the limit for items.

File: data/json/furniture_and_terrain/furniture-storage.json [hunk 4]
```diff
@@ -1137,6 +1137,29 @@
       "items": [ { "item": "scrap", "count": [ 8, 32 ] }, { "item": "water_faucet", "prob": 50 } ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_storage_tank",
+    "name": "bulk storage tank",
+    "description": "A 2 meters high tank that can store up to 1500 liters of liquid. That's a lot.",
```
Comment (GuardianDll) [inline data/json/furniture_and_terrain/furniture-storage.json:1144]:
```suggestion
    "description": "A 2 meters high tank that can store up to 1500 liters of liquid.  That's a lot.",
```

**Pull request merged**